sys/sys/atomic_common.h

```diff
@@ -65,7 +65,7 @@
     (*(volatile uint16_t *)(p) = (uint16_t)(v))
 #define	atomic_store_32(p, v)		\
     (*(volatile uint32_t *)(p) = (uint32_t)(v))
-#ifdef _LP64
+#if defined(_LP64) || __has_feature(capabilities)
```

Comment:
```suggestion
#if defined(_LP64) || __SIZEOF_LONG__ >= 64
```
might be nicer.

Reply:
Ahh, hmm, in that case we wouldn't need _LP64 at all?  Also, I think it would need to be >= 8 as the size is in bytes it seems and not bits.